### ISSUE TITLE
Broke my internet during installation. did not restore after uninstallation.

### ISSUE THREAD

**Issue by 0xorial** [complexorganizations/wireguard-manager#451]
tried on clean arch+kde.

**Comment by glebpepega** [comment]
same

**Comment by Prajwal-Koirala-DEV** [comment]
We truly appreciate everyone who has contributed by reporting bugs and helping us improve the system.

Our typical process for addressing bugs looks like this:

1. **Identify Bugs** – Gather reports and monitor for issues.
2. **Fix Bugs** – Analyze and resolve the underlying problems.
3. **Create a Pull Request** – Submit the changes for review.
4. **Test the Pull Request** – Ensure the fix works across different systems and environments.
5. **Merge the Pull Request** – Once validated, the fix is merged into the main codebase.

While these steps may seem straightforward, we know that unexpected challenges can arise, and additional issues might surface during the bug-fixing process. Despite this, your time and effort in helping us tackle these problems make a huge difference, and we're incredibly grateful for your contributions.

Thank you again for your support in making the system better for everyone!
